guides/plugins/plugins/content/cms/real-world-cms/add-real-world-cms-block.md

```diff
@@ -0,0 +1,269 @@
+# Add a real world CMS block
+
+## Overview
+
+This guide will teach you how to create a CMS-Block with your plugin. This guide will cover a real world example to keep everything easy we will just create a simple button. The CMS-block for the button we will cover right here.
```

Comment:
How close is a single button to the real world? I see it is less code to understand but how does it differ from the other guides that already exist in the documentation?

Reply:
The plan in the beginning was to also add a Data Resolver but we actually don't need it I noticed halfway through. A bit unlucky but I think this example is good to understand the connection between a cms block and a cms element because we just create both for the same "element".

In the existing guide there is a block and a element guide but they have nothing to do with each other which can be a bit confuising I guess. So maybe it's still valuable